api/v1beta1/metallb_types.go

```diff
@@ -32,6 +33,15 @@ type MetalLBSpec struct {
 	MetalLBImage string `json:"image,omitempty"`
 	// +optional
 	SpeakerNodeSelector map[string]string `json:"nodeSelector,omitempty"`
+	// tolerations is a list of tolerations applied to metalLB controller
```

Comment:
Nit: it applies to the speaker only.
I'd rename this to SpeakerTolerations (also in the json name)

Also, can you make the comment line shorter? No need to specify the default I think, as it is a list.

Reply:
nodeselector and toleration applied to daemonset which is speaker in this case do we need to put speaker in the name ? for json I didn't have speaker for nodeselector do I need to add for both ?

Reply:
We also have the controller's deployment, which they could apply to also, so users may find it a bit misleading.
Re: nodeSelector, we can't change that now.